src/Compilers/CSharp/Portable/Symbols/Source/SourceMethodSymbolWithAttributes.cs

```diff
@@ -1762,7 +1762,7 @@ internal override System.Reflection.MethodImplAttributes ImplementationAttribute
                     // PROTOTYPE: Use real value from MethodImplAttributes when available
```

Comment:
You could do this now, I think. That is, look in the ref assembly for MethodImplAttributes.Async, then write that value in.

Reply:
We don't do anything like that today. We'll just wait for the `MethodImplAttributes` that we consume to update.